### ISSUE TITLE
Tab order (keyboard navigation) in Format > Style > Rests is all over the place

### ISSUE THREAD

**Issue by cbjeukendrup** [musescore/MuseScore#25530]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

See video

### Supporting files, videos and screenshots

https://github.com/user-attachments/assets/096810f0-daba-4f40-8bac-278c608b6a4f

### What is the latest version of MuseScore Studio where this issue is present?

master

### Regression

Choose option...

### Operating system

\*

### Additional context

Allowing tab navigation into and out of the QML parts of this dialog is probably not in scope for this issue. Keyboard navigation in UIs that mix QML and QtWidgets is an unsolved problem (well, there are known workarounds, but in our case it's extra complicated because in QML we have our own keyboard nav implementation).

Should be solved in Qt Creator's design view; go to `editstyle.ui`, click the tiny "edit tab stops" button at the top of the window, and figure out how that works. Tip: right-click > "tab order list" is useful.

### Checklist

- [X] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [X] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [X] I have attached all requested files and information to this report
- [X] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible

**Comment by cbjeukendrup** [comment]
Possibly, other pages are affected too. Updating the keyboard navigation order has to be done after every edit to the dialog, but is very easy to forget.

**Comment by MarcSabatella** [comment]
FWIW, I often find it simpler to just edit the UI file directly (as text) for this type of thing.
